### PR TITLE
Add date management for X-Editable

### PR DESCRIPTION
**Pull request by netounet**
A PR to handle date field in x-editable

### CODE REVIEW

**Comment by OskarStark** [comment]
nice, can you please add some screenshots?

**Comment by netounet** [comment]
capture d ecran 2015-08-19 a 19 07 40

A little screenshot for eye pleasure :smile:

File: Controller/HelperController.php [hunk 1]
```diff
@@ -275,6 +275,11 @@ public function setObjectFieldValueAction(Request $request)
             $propertyPath = new PropertyPath($field);
         }
 
+        // Handle date type has setter expect a DateTime object
+        if($value !== '' && $fieldDescription->getType() == 'date') {
```
Comment (Soullivaneuh) [inline Controller/HelperController.php]: Yoda condition.

**Comment by Soullivaneuh** [comment]
Please fix CS: https://travis-ci.org/sonata-project/SonataAdminBundle/jobs/76277362

**Comment by OskarStark** [comment]
thank you for screenshot @netounet

**Comment by rande** [comment]
damned, I merged the code to quickly. @netounet next time squash your commit. Thanks ;)

**Comment by netounet** [comment]
@rande ok just read about squash fonctionnality :) next time

**Pull request merged**